### ISSUE TITLE
conf.py overwitten when you run build_docs.py

### ISSUE THREAD

**Issue by ryancoe** [sandialabs/WecOptTool#373]
It seems that the updates made in #347 to [build_docs.py](https://github.com/sandialabs/WecOptTool/blob/main/docs/build_docs.py) make it so that the latest version of [conf.py](https://github.com/sandialabs/WecOptTool/blob/main/docs/source/conf.py) is always used when building the docs (i.e., any changes you've made to [conf.py](https://github.com/sandialabs/WecOptTool/blob/main/docs/source/conf.py) will be overwritten). I noticed this when trying to use my workaround to avoid running the tutorials (see #335) while reviewing #372. It also has the effect of leaving your git repo in a branch other than the one you started in.

Any thoughts on this @michaelcdevin?

https://github.com/sandialabs/WecOptTool/blob/b9e1e0dd058723ba9d76f030c9c8264c1a2d8ca0/docs/build_docs.py#L56-L62

**Comment by michaelcdevin** [comment]
I discovered this too while testing the exact same thing. I was able to work around the issue by:
1. Changing [versions.yaml](https://github.com/sandialabs/WecOptTool/blob/main/docs/versions.yaml) so `'latest'` is the branch I'm interested in testing (i.e. `new_bib_refs` for #372) and commenting out all other branches
2. Doing a temporary `git commit` with those changes along with the #335 change to conf.py
3. Running build_docs.py locally to test the changes
4. Reverting the changes when I'm done testing

This is obviously very clunky and not very practical. A better solution I was thinking of was adding input arguments to build_docs.py so e.g. it only performs the `git checkout` commands you highlighted if you specify `python build_docs.py --option production` or something. Should be a pretty straightforward change
